Fixed VRRP AH extraction in pcap2john

exploide opened the pull request:
Fixed the problem described in #4653.

Using scapy is more straight forward than dpkt.

I also cleaned up the imports a bit.

exploide committed:
pcap2john: fixed VRRP AH extraction by using scapy instead of dpkt

fixes #4653

exploide committed:
pcap2john: cleanup, import ordering

solardiz commented on run/pcap2john.py:
```diff
@@ -1207,34 +1203,20 @@ def pcap_parser_ah(fname):
+            icv = pkt['AH'].icv
+            pkt['AH'].icv = bytearray(len(icv))
+            # print net-ah hash
+            print("$net-ah$0$%s$%s" % (hexlify(bytes(pkt)).decode('ascii'), hexlify(icv).decode('ascii')))
```
Looks like all other parsers use `sys.stdout.write`, not `print`. I don't really mind as long as this works across the same Python versions, although we could want to standardize on one or the other approach.

exploide replied:
`print()` should be available on more recent Python2 I think.

Agree that it should be standardized. I think `print` is a bit cleaner and more "pythonic" than `sys.stdout.write`. I changed the occurrences.

solardiz replied:
`print()` worked for me in a quick Python shell test with RHEL6's 2.6.6.

exploide committed:
pcap2john: cleanup; use print instead of sys.stdout.write; moved message

exploide committed:
pcap2john: fixed vrrp parsing

didn't print anything with python3
it's now compatible with python2 and python3
simplyfied parsing using scapy

exploide commented:
The other (non-AH) VRRP packets were also not correctly parsed with python3. Fixed that function, too.

It's amazing how simple the parsing becomes when using scapy.
